books/free-programming-books-langs.md

```diff
@@ -1981,6 +1981,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Learn Python the Right Way](https://learnpythontherightway.com)
 * [Learn Python Visually](https://archive.org/details/learn-python-visually_compress/mode/2up) - Ivelin Demirov *(:card_file_box: archived)*
 * [Learn to Program Using Python](https://web.archive.org/web/20201224032210/https://www.ida.liu.se/~732A47/literature/PythonBook.pdf) - Cody Jackson (PDF) *(:card_file_box: archived)*
+*[Learning Python](https://cfm.ehu.es/ricardo/docs/python/Learning_Python.pdf)
```

Comment:
This distribution is unauthorized. Please check docs/CONTRIBUTING.md for guidelines and respect the Markdown formatting in the files